hera_cal/redcal.py

```diff
@@ -297,37 +486,37 @@ def build_eqs(self, bls_in_data):
                     eqs['g%d%s * g%d%s_ * u%d%s' % params] = (ant_i, ant_j, pol)
         return eqs
 
-    def _solver(self, solver, data, wgts={}, detrend_phs=False, sparse=False, **kwargs):
+    def _solver(self, solver, data, wgts={}, detrend_phs=False, **kwargs):
         """Instantiates a linsolve solver for performing redcal.
 
         Args:
             solver: linsolve solver (e.g. linsolve.LogProductSolver or linsolve.LinProductSolver)
             data: visibility data in the dictionary format {(ant1,ant2,pol): np.array}
             wgts: dictionary of linear weights in the same format as data. Defaults to equal wgts.
             detrend_phs: takes out average phase, useful for logcal
-            sparse: represent the A matrix (visibilities to parameters) sparsely in linsolve
             **kwargs: other keyword arguments passed into the solver for use by linsolve
```

Comment:
I see why you removed sparse, but I'd add a note to the kwargs like (e.g. "sparse")

Reply:
Done.